sdk/python/tests/integration/feature_repos/universal/entities.py

```diff
@@ -0,0 +1,8 @@
+from feast import Entity, ValueType
+
+driver = Entity(
```

Comment:
Nit: this is just an `Entity` defined here, but in `universal/feature_views.py` we have a function `correctness_feature_view` that returns a `FeatureView`. We should standardize this.